go/serialization.go

```diff
@@ -151,6 +151,33 @@ func ReadTimestampedEntryInto(r io.Reader, t *TimestampedEntry) error {
 	return nil
 }
 
+// SerializeTimestampedEntry writes timestamped entry to Writer.
+func SerializeTimestampedEntry(w io.Writer, t *TimestampedEntry) error {
+	if err := binary.Write(w, binary.BigEndian, t.Timestamp); err != nil {
```

Comment:
Does go allow you to concatenate the condition here and in line 159? i.e.
if err:=...; err != nill || err:=...; err != nill {
  return err
}

?